src/amber/pipes/static.cr

```diff
@@ -10,7 +10,10 @@ module Amber
       def call(context : HTTP::Server::Context)
         return allow_get_or_head(context) unless method_get_or_head?(context.request.method)
 
-        original_path = context.request.path.not_nil!
+        scope = ""
+        context.request.matched_route.payload.try { |r| r.scope.to_s }
```

Comment:
This line seems complicated, and not very Demeter friendly.

Add methods that helps comprehension.

Reply:
I rewrites and fixed it here in new commit.